data/projects/coala_stats.md

```diff
@@ -0,0 +1,25 @@
+Statistics like :
+
+#### Normal stats
+- How many times was coala ran? 
+
+#### Data regarding coala patches
+This would give us insights into how many patches suggested by coala actually
+turn out to be correct and how many are not. Accordingly, eventually we may
+be able to propose patches with accurate confidence.
+
+- How many patches did coala suggest to the user? 
+- How many of those patches were accepted by the user? 
+- How many of the patches proposed by coala were edited by the user? 
+- How many patches/errors were ignored/silenced? (Incorrect errors...)
+
+#### Misc. 
+- Repository specific statistics, Bear specific statistics and global statistics.
+- How many changes did coala bring to the source code? 
+
+#### Using the data for promotion
+- Option to permit coala transmit these data to the coala server. 
+- Showcase the visualisations of the data on the coala website. 
+
+Data visualisations are a good means of promotions and also to prove the power of coala(and help it conquer the world eventually.)
```

Comment:
nitpick, line seems too long.
Do we follow certain style guide for this?
If not, ignore.